simapp/v2/app_di.go

```diff
@@ -255,3 +258,7 @@ func (app *SimApp[T]) GetConsensusAuthority() string {
 func (app *SimApp[T]) GetStore() any {
 	return app.App.GetStore()
 }
+
+func (app *SimApp[T]) GetGRPCQueryDecoders() map[string]func(requestBytes []byte) (gogoproto.Message, error) {
+	return app.GRPCQueryDecoders
+}
```

Comment:
anyway to avoid this? seems the value is public so we should be able to get it directly

Reply:
let's have runtime expose it instead of simapp indeed